components/SearchSelect/SearchSelect.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { FunctionComponent } from 'react';
```

Comment:
Avoid repetition ... Drop this component and add a `prop` to the existing `Select` component... call it `theme` with two possible values `dark` & `light`. `light` is the default. Change the style as required for the dark option.

Reply:
`chooseTheme` option with either dark or light toggles between a location select and a normal select.